src/enums/enums.months.js

```diff
@@ -0,0 +1,5 @@
+module.exports = ['January', 'February', 'March', 
+'April', 'May', 'June', 'July', 'August', 'September', 
+'October', 'November', 'December', 'Jan', 'Feb', 'Mar',
```

Comment:
Whats the reason for supported 2 different formats?

Reply:
we'll remove later if that comes to it.

Reply:
Talked offline -> he says it is good.